Codigo/General.bas

```diff
@@ -46,42 +46,56 @@ Global LeerNPCs As New clsIniManager
 
 
 Sub SetNakedBody(ByRef user As t_User)
+        Const man_human_naked_body As Integer = 21
+        Const man_drow_naked_body As Integer = 32
+        Const man_elf_naked_body As Integer = 510
+        Const man_gnome_naked_body As Integer = 508
+        Const man_dwarf_naked_body As Integer = 508
+        Const man_orc_naked_body As Integer = 53
```

Comment:
this values doesn't match prev hardcoded values,  508 is duplicated and the orc is using dwarf values

Reply:
Fixed in last commit!

Good catch @matiascalegaris !